docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -21,7 +21,7 @@ To use this guide, you must first [create an Azure storage account](/azure/stora
 
 The samples in this article can be used in either an F# application or an F# script. To create an F# script, create a file with the `.fsx` extension, for example `blobs.fsx`, in your F# development environment.
 
-Next, use a [package manager](package-management.md) such as [Paket](https://fsprojects.github.io/Paket/) or [NuGet](https://www.nuget.org/) to install the `WindowsAzure.Storage` and `Microsoft.WindowsAzure.ConfigurationManager` packages and reference `WindowsAzure.Storage.dll` and `Microsoft.WindowsAzure.Configuration.dll` in your script using a `#r` directive.
+Next, use a [package manager](package-management.md) such as [Paket](https://fsprojects.github.io/Paket/) or [NuGet](https://www.nuget.org/) to install the `Azure.Storage.Blobs` and `Azure.Storage.Blobs.Models` and `Azure.Storage.Blobs.Specialized` packages and reference `Azure.Storage.Blobs.dll` in your script using a `#r` directive.
```

Comment:
Probably out of scope here, but the package manager article is out of date. It should really show how to use `#r "nuget:..."` to quickly get to nuget packages - this is the recommended "defacto" way today to get to nuget packages in a script.